.github/workflows/test.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
```

Comment:
Changed here because CI should be run in any PRs 
@hirosassa

Reply:
👍